projects/ngx-charts-on-fhir/src/lib/fhir-data/fhir-data.service.ts

```diff
@@ -94,6 +99,86 @@ export class FhirDataService {
       return teardownLogic;
     });
   }
+
+  addPatientData(resource: Meta) {
```

Comment:
I doubt that [Meta](https://hl7.org/fhir/resource.html#Meta) is the correct type to use here. Please take another look at this.